aptible-auth.gemspec

```diff
@@ -23,11 +24,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'gem_config'
   spec.add_dependency 'oauth2-aptible', '~> 0.10.0'
 
+  spec.add_development_dependency 'aptible-tasks', '>= 0.6.0'
   spec.add_development_dependency 'bundler', '~> 1.3'
-  spec.add_development_dependency 'aptible-tasks', '>= 0.2.0'
+  spec.add_development_dependency 'pry'
```

Comment:
Q: Is this change to ensure alphabetical ordering? If so, I love it.

Reply:
also rubocop